Makefile

```diff
@@ -111,7 +126,7 @@ docs/%.8: docs/%.8.md check-gopath
 
 docs: $(MANPAGES)
 
-install: check-gopath
+install: ocid ocic kpod conmon/conmon pause/pause
```

Comment:
We have gone back and forth on this and I think we don't want to depend on the binaries in install.